core/src/main/java/com/oath/oak/ValueBuffer.java

```diff
@@ -0,0 +1,32 @@
+package com.oath.oak;
+
+public class ValueBuffer extends OakAttachedReadBuffer {
+    protected long reference;
```

Comment:
It is accessed directly. `ValueBuffer` was a subclass of `EntrySet`, so this was OK to access its internal members. Now it will be better to add getter/setter.

Reply:
Fixed.